Fromm: Luca Bruno <lucab@debian.org>
Description: Misc fixes to build libgit2-sys
Forwarded: not-needed
--- a/deps/libgit2-sys-0.4.3/build.rs
+++ b/deps/libgit2-sys-0.4.3/build.rs
@@ -102,7 +102,6 @@
         cfg.define("USE_OPENSSL", "OFF");
     }
 
-    let _ = fs::remove_dir_all(env::var("OUT_DIR").unwrap());
     t!(fs::create_dir_all(env::var("OUT_DIR").unwrap()));
 
     let dst = cfg.define("BUILD_SHARED_LIBS", "OFF")
